diff -Naur clamav-0.96/clamav-milter/Makefile.am clamav-0.96.oden/clamav-milter/Makefile.am
--- clamav-0.96/clamav-milter/Makefile.am	2009-07-06 15:21:58.000000000 +0200
+++ clamav-0.96.oden/clamav-milter/Makefile.am	2010-04-01 11:37:38.666295974 +0200
@@ -46,6 +46,6 @@
 
 DEFS = @DEFS@ -DCL_NOLIBCLAMAV
 CFLAGS=`echo "@CFLAGS@" | sed -e 's/-Wwrite-strings//' -e 's/-Werror /-Werror -Wno-error=format-nonliteral /'`
-LIBS = $(top_builddir)/libclamav/libclamav_internal_utils.la @CLAMAV_MILTER_LIBS@ @THREAD_LIBS@
+LIBS = @CLAMAV_MILTER_LIBS@ $(top_builddir)/libclamav/libclamav_internal_utils.la @THREAD_LIBS@
 AM_CPPFLAGS = -I$(top_srcdir)/clamd -I$(top_srcdir)/libclamav -I$(top_srcdir)/shared -I$(top_srcdir)
 CLEANFILES=*.gcda *.gcno
diff -Naur clamav-0.96/clamav-milter/Makefile.in clamav-0.96.oden/clamav-milter/Makefile.in
--- clamav-0.96/clamav-milter/Makefile.in	2010-03-24 18:01:18.000000000 +0100
+++ clamav-0.96.oden/clamav-milter/Makefile.in	2010-04-01 11:37:38.666295974 +0200
@@ -213,7 +213,7 @@
 LIBLTDL = @LIBLTDL@
 LIBM = @LIBM@
 LIBOBJS = @LIBOBJS@
-LIBS = $(top_builddir)/libclamav/libclamav_internal_utils.la @CLAMAV_MILTER_LIBS@ @THREAD_LIBS@
+LIBS = @CLAMAV_MILTER_LIBS@ $(top_builddir)/libclamav/libclamav_internal_utils.la @THREAD_LIBS@
 LIBTOOL = @LIBTOOL@
 LIPO = @LIPO@
 LN_S = @LN_S@
