diff -up clamav-0.102.0/freshclam/freshclam.c clamav-0.102.0/freshclam/freshclam.c-angry
--- clamav-0.102.0/freshclam/freshclam.c        2019-10-01 19:24:08.000000000 +0200
+++ clamav-0.102.0/freshclam/freshclam.c-angry  2019-11-02 02:14:31.667081297 +0100
@@ -124,7 +124,7 @@ static void writepid(const char *pidfile
 {
     FILE *fd;
     int old_umask;
-    old_umask = umask(0006);
+    old_umask = umask(0022);
     if ((fd = fopen(pidfile, "w")) == NULL) {
         logg("!Can't save PID to file %s: %s\n", pidfile, strerror(errno));
     } else {
diff -up clamav-0.102.0/shared/output.c clamav-0.102.0/shared/output.c-angry
--- clamav-0.102.0/shared/output.c      2019-10-01 19:24:09.000000000 +0200
+++ clamav-0.102.0/shared/output.c-angry        2019-11-02 02:15:53.914082618 +0100
@@ -359,7 +359,7 @@ int logg(const char *str, ...)
     logg_open();
 
     if (!logg_fp && logg_file) {
-        old_umask = umask(0037);
+        old_umask = umask(0077);
         if ((logg_fp = fopen(logg_file, "at")) == NULL) {
             umask(old_umask);
 #ifdef CL_THREAD_SAFE
