diff -up clamav-0.100.0/freshclam/freshclam.c.0013~ clamav-0.100.0/freshclam/freshclam.c
--- clamav-0.100.0/freshclam/freshclam.c.0013~	2018-04-04 02:13:58.000000000 +0200
+++ clamav-0.100.0/freshclam/freshclam.c	2018-06-20 21:42:59.407239234 +0200
@@ -127,7 +127,7 @@ writepid (const char *pidfile)
 {
     FILE *fd;
     int old_umask;
-    old_umask = umask (0006);
+    old_umask = umask (0022);
     if ((fd = fopen (pidfile, "w")) == NULL)
     {
         logg ("!Can't save PID to file %s: %s\n", pidfile, strerror (errno));
diff -up clamav-0.100.0/shared/output.c.0013~ clamav-0.100.0/shared/output.c
--- clamav-0.100.0/shared/output.c.0013~	2018-06-20 21:42:59.407239234 +0200
+++ clamav-0.100.0/shared/output.c	2018-06-20 21:43:29.503237576 +0200
@@ -379,7 +379,7 @@ int logg(const char *str, ...)
 
     if (!logg_fp && logg_file)
     {
-        old_umask = umask(0037);
+        old_umask = umask(0077);
         if ((logg_fp = fopen(logg_file, "at")) == NULL)
         {
             umask(old_umask);
