diff -Naur clamav-0.97.5/clamav-milter/Makefile.am clamav-0.97.5.oden/clamav-milter/Makefile.am
--- clamav-0.97.5/clamav-milter/Makefile.am	2012-06-17 09:59:20.989781346 +0200
+++ clamav-0.97.5.oden/clamav-milter/Makefile.am	2012-06-17 09:59:37.584632940 +0200
@@ -45,7 +45,7 @@
 endif
 
 DEFS = @DEFS@ -DCL_NOLIBCLAMAV
-CFLAGS=`echo "@CFLAGS@" | sed -e 's/-Wwrite-strings//' -e 's/-Werror /-Werror -Wno-error=format-nonliteral /'`
+CFLAGS=@CFLAGS@
 LIBS = @CLAMAV_MILTER_LIBS@ $(top_builddir)/libclamav/libclamav_internal_utils.la @THREAD_LIBS@
 AM_CPPFLAGS = -I$(top_srcdir)/clamd -I$(top_srcdir)/libclamav -I$(top_srcdir)/shared -I$(top_srcdir)
 CLEANFILES=*.gcda *.gcno
diff -Naur clamav-0.97.5/clamav-milter/Makefile.in clamav-0.97.5.oden/clamav-milter/Makefile.in
--- clamav-0.97.5/clamav-milter/Makefile.in	2012-06-17 09:59:20.989781346 +0200
+++ clamav-0.97.5.oden/clamav-milter/Makefile.in	2012-06-17 09:59:37.585632932 +0200
@@ -167,7 +167,7 @@
 CC = @CC@
 CCDEPMODE = @CCDEPMODE@
 CFGDIR = @CFGDIR@
-CFLAGS = `echo "@CFLAGS@" | sed -e 's/-Wwrite-strings//' -e 's/-Werror /-Werror -Wno-error=format-nonliteral /'`
+CFLAGS = @CFLAGS@
 CHECK_CPPFLAGS = @CHECK_CPPFLAGS@
 CHECK_LIBS = @CHECK_LIBS@
 CLAMAVGROUP = @CLAMAVGROUP@
