--- clamav-0.97.2/libclamav.pc.in	2011-07-14 13:14:01.000000000 +0200
+++ clamav-0.97.2.dlucio/libclamav.pc.in	2011-07-25 20:53:52.000000000 +0200
@@ -6,6 +6,7 @@
 Name: libclamav
 Description: A GPL virus scanner
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -lclamav @LIBCLAMAV_LIBS@
-Cflags: -I${includedir} @CFLAGS@
+Libs: -L${libdir} -lclamav
+Libs.private: -L${libdir} -lclamav @LIBCLAMAV_LIBS@
+Cflags: -I${includedir}
 
--- clamav-0.97.2/clamav-config.in	2011-01-10 18:48:28.000000000 +0100
+++ clamav-0.97.2.dlucio/clamav-config.in	2011-07-25 20:53:43.000000000 +0200
@@ -54,14 +54,9 @@
 	usage 0
 	;;
 
-    --cflags)
-       	echo -I@includedir@ @CFLAGS@
-       	;;
-
-    --libs)
-       	echo -L@libdir@ @LIBCLAMAV_LIBS@
-       	;;
-
+    (--cflags|--libs)
+               ${PKG_CONFIG:-pkg-config} "$1" libclamav
+                ;;
     *)
 	usage
 	exit 1
